src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs

```diff
@@ -114,6 +114,18 @@ public void TexlParseNumericLiterals_Negative(string script)
         public void TexlParseLargeNumerics_Negative(string script)
         {
             TestParseErrors(script, 1, StringResources.Get(TexlStrings.ErrNumberTooLarge));
+        }
+
+        [Theory]
+        [InlineData("[{A]", 0, 4)]
+        [InlineData("[{A:2}]", 0, 7)]
+        [InlineData("With({A:2", 0, 9)]
+        public void TexlParseRecordNodes(string script, int min, int lim)
```

Comment:
>TexlParseRecordNodes [](http://example.com/codeflow?start=20&length=20)

[minor] rename to TestParseRecordNodesSpan - and move closer to tests for record nodes (TestParseRecords)